src/lib/api/contrib/invenio-records-resources/InvenioRecordsResourcesRequestSerializer.js

```diff
@@ -0,0 +1,110 @@
+/*
```

Comment:
Can you please move this into the `invenio` contrib? It is just a new request serializer, not an entire new `contrib`.